gpytorch/models/exact_prediction_strategies.py

```diff
@@ -651,47 +650,7 @@ def exact_predictive_covar(self, test_test_covar, test_train_covar):
 
 
 class SumPredictionStrategy(DefaultPredictionStrategy):
```

Comment:
Perhaps we should just get rid of `SumPredictionStrategy` at this point, if it's going to be a no-op? I am open to this: this code is old and stale, and it's probably best to use the default prediction strategy at this point.

Reply:
Sound good to me, will adjust